src/mock/api/rules.ts

```diff
@@ -0,0 +1,13 @@
+export function rulesMock(flow: string, { executedRules = [] }: { executedRules?: string[] } = {}) {
+  const state = {
+    executed: executedRules,
+  };
```

Comment:
Let's not put this in the state for now. It's only a value you can set in your preconditions. An Action can't modify this, so there will never be a need to test it in postconditions.